lib/conjur/api/roles.rb

```diff
@@ -19,9 +19,32 @@
 # CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #
 require 'conjur/role'
+require 'conjur/graph'
 
 module Conjur
   class API
+    ##
+    # Fetch a digraph (or a forest of digraphs) representing of
+    #   role memberships related transitively to any of a list of roles.
+    #
+    # @param [Array<Conjur::Role, String>] roles the  digraph (or forest thereof) of
+    #   the ancestors and descendants of these roles or role ids will be returned
```

Comment:
BTW, @kgilpin , @jjmason , I was thinking -- what was the rationale for transitive descendant visibility? It seems overly lax and I couldn't think of why it's needed after I added 'skin' visibility to the graph, but I believe I wasn't following the original discussion so I could have missed something. Failing that, maybe we should drop it before it becomes released and entrenched?